### ISSUE TITLE
Pebble Templates not rendering model

### ISSUE THREAD

**Issue by cosmin-marginean** [http4k/http4k#980]
It seems that the Pebble templates wiring is broken as it can't render model data.
To isolate this I've forked the [examples repo](https://github.com/http4k/examples) repo and replaced the `web-content` example wiring for Handlebars with Pebble (PR [here](https://github.com/http4k/examples/pull/222/files) for ease of navigation). The result was that the dynamic content wasn't being rendered anymore. Below a screenshot with the before and after.

<img width="2129" alt="2023-10-05_22-37-23" src="https://github.com/http4k/http4k/assets/2995576/860d0120-731d-4fab-9fa1-ba07f22a60cb">

To reproduce this simply:
1. Check out the forked repo (https://github.com/cosmin-marginean/http4k-examples)
2. Run `./gradlew run` from the `web-content` directory
3. Open http://localhost:8080/ in the browser.

**Comment by cosmin-marginean** [comment]
It turns out this is actually a documentation problem. Unlike other template engines that can be used with http4k, Pebble is going to access the model via `model.field`, as opposed to simply `field`.

I only figured this out once I looked at the code for how Pebble Templates are wired: https://github.com/http4k/http4k/blob/master/http4k-template/pebble/src/main/kotlin/org/http4k/template/PebbleTemplates.kt#L15 and it wasn't obvious anywhere that this is necessary.

I believe the docs need to be updated to reflect these types of subtleties.
